HelpSource/Classes/String.schelp

```diff
@@ -248,18 +248,21 @@ code::
 ::
 
 method::asSecs
-Return a link::Classes/Float:: based on converting a time string in format (ddd):hh:mm:ss.sss. This is the inverse method to link::Classes/SimpleNumber#-asTimeString::.
+Return a link::Classes/Float:: based on converting a time string in format (ddd:)hh:mm:ss(.z), where z is any number of digits. This is the inverse method to link::Classes/SimpleNumber#-asTimeString::.
```

Comment:
"sequence of digits" would be more accurate, i think. also, can you wrap the format string in `code:: ::` to match the documentation for `asTimeString`?